h2o-docs/src/product/data-science/pca.rst

```diff
@@ -11,51 +11,67 @@ PCA is commonly used to model without regularization or perform dimensionality r
 
--  `seed <algo-params/seed.html>`__: Specify the random number generator (RNG) seed for algorithm components dependent on randomization. The seed is consistent for each H2O instance so that you can create models with the same starting conditions in alternative configurations. This value defaults to -1 (time-based random number).
+-  `export_checkpoints_dir <algo-params/export_checkpoints_dir.html>`__: Specify a directory to which generated models will automatically be exported.
 
--  `use_all_factor_levels <algo-params/use_all_factor_levels.html>`__: Specify whether to use all factor levels in the possible set of predictors; if you enable this option, sufficient regularization is required. By default, the first factor level is skipped. For PCA models, this option ignores the first  factor level of each categorical column when expanding into indicator columns. This option is disabled by default.
+Hyperparameters
+'''''''''''''''
 
--  `compute_metrics <algo-params/compute_metrics.html>`__: Enable metrics computations on the training data. This option is enabled by default.
+These parameters can be used in grid search.
 
--  `score_each_iteration <algo-params/score_each_iteration.html>`__: (Optional) Specify whether to score during each iteration of the model training. This option is disabled by default.
+-  `transform <algo-params/transform.html>`__: Specify the transformation method for numeric columns in the training data. One of: 
 
--  `max_runtime_secs <algo-params/max_runtime_secs.html>`__: Maximum allowed runtime in seconds for model training. This value is set to 0 (disabled) by default.
+  - ``none`` (default)
+  - ``standardize``
+  - ``normalize``
+  - ``demean``
+  - ``descale`` 
 
--  `impute_missing <algo-params/impute_missing.html>`__: Specifies whether to impute missing entries with the column mean value. This value defaults to False.
+-  `k <algo-params/k.html>`__: Specify the rank of matrix approximation. This can be a value from 1 to the minimum of (total number of rows, total number of columns) in the dataset. This value defaults to ``1``.
 
--  `export_checkpoints_dir <algo-params/export_checkpoints_dir.html>`__: Specify a directory to which generated models will automatically be exported.
+-  `max_iterations <algo-params/max_iterations.html>`__: Specify the number of training iterations. The value must be between 1 and 1e6 and the default is ``1000``.
+
+-  `compute_metrics <algo-params/compute_metrics.html>`__: Enable metrics computations on the training data. This option defaults to ``True`` (enabled).
+
+-  `seed <algo-params/seed.html>`__: Specify the random number generator (RNG) seed for algorithm components dependent on randomization. The seed is consistent for each H2O instance so that you can create models with the same starting conditions in alternative configurations. This value defaults to ``-1`` (time-based random number).
+
```

Comment:
seed is not gridable

Reply:
I got scared that we have a bug in SE (we have `seed` as gridable) but after short inspection I found that `seed` is usually gridable except for PCA, SVD, and Aggregator. I'm mentioning this just to avoid confusion.